packages/web-twig/src/Resources/components/Tooltip/stories/TooltipPlacements.twig

```diff
@@ -1,44 +1,43 @@
-<form autocomplete="off" class="mx-auto">
-    <Grid cols="3" UNSAFE_style="align-items: center; justify-items: center">
+<form autocomplete="off">
+    <Grid cols="3" UNSAFE_className="mx-auto" UNSAFE_style="align-items: center; justify-items: center; max-width: 30rem;">
         <div style="grid-row: 1; grid-column: 2">
 
-            <Radio name="placement" isLabelHidden value="topLeft" id="placement_top_left" label="top-left" />
-            <Radio name="placement" isLabelHidden value="top" id="placement_top" label="top" />
-            <Radio name="placement" isLabelHidden value="topRight" id="placement_top_right" label="top-right" />
+            <Radio name="placement" islabelhidden value="topLeft" id="placement_top_left" label="top-left"/>
```

Comment:
Not a blocker, just curious why are `islabelhidden` and `ischecked`  lowercased?

Reply:
Because, I screwed it 🤦‍♂️

Reply:
Fixed. Thanks for noticing.